scripts/config

```diff
@@ -77,14 +103,17 @@ apply_config() {
 
 	# Set overwrite_nginx
 	overwrite_nginx=$(bool_to_01 $overwrite_nginx)
-	ynh_app_setting_set $app overwrite_nginx "$overwrite_nginx"
+	ynh_app_setting_set --app=$app --key=overwrite_nginx --value="$overwrite_nginx"
 	# Set overwrite_phpfpm
 	overwrite_phpfpm=$(bool_to_01 $overwrite_phpfpm)
-	ynh_app_setting_set $app overwrite_phpfpm "$overwrite_phpfpm"
+	ynh_app_setting_set --app=$app --key=overwrite_phpfpm --value="$overwrite_phpfpm"
 
 	# Set admin_mail_html
 	admin_mail_html=$(bool_to_01 $admin_mail_html)
-	ynh_app_setting_set $app admin_mail_html "$admin_mail_html"
+	ynh_app_setting_set --app=$app --key=admin_mail_html --value="$admin_mail_html"
+
+	# Reconfigure php-fpm
+	config-panel_scripts/configure_php_fpm --fpm_usage=$fpm_usage --fpm_footprint=$fpm_footprint
```

Comment:
Can you explain to me why did you split this part?

Reply:
Because, to not overload the config script with to much specific code, I rather use a dedicated script. Like I did with the public/private setting.
Even though the script is not that big, it seems clearer to have dedicated script for each actions.

I could have use an action, like public/private, but in that case that means having an actual action for it, which wouldn't work without arguments.
Also, as I was about to work on standard config-panel and actions to add to the example app, it easier to have separate scripts for each thing, so a packager could easily build his own config-panel/action by selecting what it would like to add to his app.

Reply:
Don't you think this it's not too much confusing for lambda packager?

Plus, we fight to not split too much the source code in "standards scripts", it's little weird to do it in config panel, don't you think?

(I'm not against the idea, just wondering)

Reply:
Actually, it's on the same idea.
My idea here is to have standard scripts for common actions, public/private, php config, and probably other things later
My idea is to not overload the config script itself to not have a unreadable script.

But maybe indeed, we could rely on parts, like other scripts to do it.

I was thinking of this way because of my idea to provide built script, ready to be used.
But anyway, we'll need to add parts into the json (soon toml) and the config script itself.
So maybe we should rather keep everything into the same place, and just use separate scripts for actions, which don't have a common script.

Reply:
I moved the script back to 'config' instead of a dedicated script.